app/src/main/java/org/mozilla/tv/firefox/channels/content/SportsChannels.kt

```diff
@@ -24,13 +24,5 @@ fun ChannelContent.getSportsChannels(): List<ChannelTile> = listOf(
         setImage = setImage(R.drawable.tile_sports_formula_1),
         tileSource = TileSource.SPORTS,
         id = "formula1"
-    ),
-    ChannelTile(
```

Comment:
nit: in the commit message, you should mention which commit this is roughly backing out.